### PR TITLE
Modern theme styles for dropdown

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-tab-dropdown-modern-theme-states](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-tab-dropdown-modern-theme-states)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyux/popovers",
-  "version": "4.1.1",
+  "version": "4.0.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json:3]: Downgrade?

Comment (Blackbaud-PaulCrowder) [reply package.json:3]: Yikes, good catch. That must have come from a bad merge.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-tab-dropdown-modern-theme-states](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-tab-dropdown-modern-theme-states)

**Pull request merged**